test-material/README.md

```diff
@@ -15,9 +15,9 @@
     - [Ranges](#ranges)
       - [Cooler List Functions](#cooler-list-functions)
   3. [List Comprehensions (WIP)](#list-comprehensions)
```

Comment:
Could you change this so it would be 
```diff
- [List Comprehensions (WIP)](#list-comprehensions)
+ 3. [List Comprehensions](#list-comprehensions)
```